packages/synthchron-tauri/src/components/editor/bottomDrawer/configurationPanel/MinMaxSlider.tsx

```diff
@@ -72,7 +76,7 @@ export const MinMaxSlider: React.FC<MinMaxSliderProps> = ({
           }
           valueLabelDisplay='auto'
           min={0}
-          max={1000}
+          max={10000}
```

Comment:
the max is just what is displayed to the user, in the input field u can insert larger numbers. the slider should have a value that makes dragging the slider meaningful. i suggest keeping 10000 as max, but changing steps to be in 10s or 100s

Reply:
That's fair! When testing with different process models I just found that a maximum of 1000 was usually way too low and I always had to use the number field. This felt a bit easier. I was also thinking of making the scale exponential, or at least polynomial. The different between 10 and 100 simulations should be similar to the difference between 1000 and 10.000